ext/winevt/winevt_channel.c

```diff
@@ -63,7 +63,7 @@ rb_winevt_channel_each(VALUE self)
     winevtChannel->channels = hChannels;
   } else {
     _snprintf_s(
-      errBuf, 256, _TRUNCATE, "Failed to enumerate channels with %s\n", GetLastError());
```

Comment:
We should use `sizeof(errBuf)` instead of 256?

Reply:
Ah, yes.

Reply:
We'll use `_countof()` which can be used in most cases both ANSI and UNICODE API.
I'll do it in another PR.

Reply:
I committed it directly to the master branch.